eng/Version.Details.xml

```diff
@@ -36,7 +37,8 @@
       <Uri>https://github.com/dotnet/corefx</Uri>
       <Sha>1719a3fe2a5c81b67a4909787da4a02fb0d0d419</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App" Version="3.0.0-preview7-27912-14" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
+    <!-- fake version number to fix version rollback, will be fixed later -->
```

Comment:
File issue to track?

Reply:
I believe I've fixed this issue and the workaround isn't needed anymore.